.vsconfig

```diff
@@ -1,53 +1,55 @@
 {
   "version": "1.0",
   "components": [
-    "Microsoft.VisualStudio.Component.Roslyn.Compiler",
-    "Microsoft.VisualStudio.Component.Roslyn.LanguageServices",
+    "Component.CPython39.x64",
+    "Microsoft.Component.CodeAnalysis.SDK",
     "Microsoft.Component.MSBuild",
-    "Microsoft.Net.Component.4.5.2.TargetingPack",
-    "Microsoft.VisualStudio.Component.SQL.LocalDB.Runtime",
-    "Microsoft.VisualStudio.Component.SQL.CLR",
-    "Microsoft.VisualStudio.Component.CoreEditor",
-    "Microsoft.VisualStudio.Workload.CoreEditor",
-    "Microsoft.Net.Component.4.8.SDK",
-    "Microsoft.Net.Component.4.7.2.TargetingPack",
-    "Microsoft.Net.ComponentGroup.DevelopmentPrerequisites",
-    "Microsoft.VisualStudio.Component.TextTemplating",
-    "Microsoft.VisualStudio.Component.NuGet",
-    "Microsoft.VisualStudio.Component.ManagedDesktop.Core",
-    "Microsoft.NetCore.Component.SDK",
-    "Microsoft.VisualStudio.Component.FSharp",
+    "Microsoft.ComponentGroup.Blend",
```

Comment:
Do you know why we pull in blend? I see we had it before... It just feels wasteful.

Reply:
I don't know why we pull in blend. I'll try a build without it.

Reply:
The build succeeds without blend. It's only 1 MB (according to the installer UI), but removing it also removes the shortcut from the start menu, so that's cool. I'll remove it from the list.